docs/quickstart/aws_cli.md

```diff
@@ -16,7 +16,7 @@ In this section we'll review how to copy files into lakeFS using the AWS CLI.
    ```bash
    aws configure --profile local
    # output:
-   # AWS Access Key ID [None]: AKIAIOSFODNN7EXAMPLE
+   # AWS Access Key ID [None]: AKIAJVHTOKZWGCD2QQYQ
```

Comment:
Does this keep the example access key consistent with other sections, or did you prefer a more realistic access key example that doesn't end with "EXAMPLE"?

Reply:
It keeps it consistent!